src/NServiceBus.Core/Recoverability/FirstLevelRetries/ProcessingFailureInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus
+{
+    using System;
+
+    class ProcessingFailureInfo
```

Comment:
What if a satellite like the gateway wants to use it? would we make it public? (I'd say no)